pkgs.nix

```diff
@@ -1,6 +1,6 @@
 let
-  sha256 = "sha256:1wg55jlxyvbjvm8x2rcirmvqws4y8xq504dn3yjp05m1bajhpj5r";
-  rev = "6daa4a5c045d40e6eae60a3b6e427e8700f1c07f";
+  sha256 = "sha256:04855jws2wwl31cbdgx15bs2nwwv3b6rz9bzh9xcx5znkyiminfr";
```

Comment:
not sure if this will break other devices support

@samueldr when should this be updated?

Reply:
Whenever, but should be verified to work.

Reply:
A fresher update is in https://github.com/NixOS/mobile-nixos/pull/480, if @tomfitzhenry doesn't end up rebasing, I'll handle it (and obviously test against that).